notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.ipynb

```diff
@@ -18,7 +18,28 @@
    "cell_type": "markdown",
```

Comment:
<p>Thanks for your work! I wanted to bring your attention to a small detail: there seems to be an issue with the prior WFPCS <code>example</code>&nbsp;link.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>